### PR TITLE
Example for the demo project and readme

### PR DESCRIPTION
**Pull request by dostrander**

### CODE REVIEW

File: Pod/Classes/RZIntrinsicContentSizeTextView.m [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+    [self adjustPlaceholderPosition];
+    [self scrollToBottom];
+}
+
+- (void)setText:(NSString *)text
+{
+    [super setText:text];
+    [self adjustPlaceholderForTextChange];
+    [self adjustHeightIfNeededAnimated:self.shouldAnimateSizeChange];
+
```
Comment (jatraiz) [inline Pod/Classes/RZIntrinsicContentSizeTextView.m]: extra \n

File: README.md [hunk 2]
```diff
@@ -1 +1,137 @@
 # RZIntrinsicContentSizeTextView
+
+<!--[![CI Status](http://img.shields.io/travis/Derek Ostrander/RZIntrinsicContentSizeTextView.svg?style=flat)](https://travis-ci.org/Derek Ostrander/RZIntrinsicContentSizeTextView)
+[![Version](https://img.shields.io/cocoapods/v/RZIntrinsicContentSizeTextView.svg?style=flat)](http://cocoadocs.org/docsets/RZIntrinsicContentSizeTextView)
+[![License](https://img.shields.io/cocoapods/l/RZIntrinsicContentSizeTextView.svg?style=flat)](http://cocoadocs.org/docsets/RZIntrinsicContentSizeTextView)
+[![Platform](https://img.shields.io/cocoapods/p/RZIntrinsicContentSizeTextView.svg?style=flat)](http://cocoadocs.org/docsets/RZIntrinsicContentSizeTextView)
+-->
+RZIntrinsicContentSizeTextView is a convinient UITextView subclass that has all the features you wish were in UITextView: a placeholder and the ability to dynamically grow based on it's content size.
+
```
Comment (jatraiz) [inline README.md:9]: spelling : convenient

File: README.md [hunk 3]
```diff
@@ -1 +1,137 @@
...
+This is ideal for the many messaging applications where the user is typing into a textview that will start off small.
+
```
Comment (jatraiz) [inline README.md:11]: , but will then need to grow in height as needed

File: README.md (hunk 2 above)
Comment (ZevEisenberg) [inline README.md:9]: spelling: “it's” → “its”

File: README.md [hunk 4]
```diff
@@ -1 +1,137 @@
...
+The placeholder interface is similar to that of a UITextField.   
+
+	.placeholder text of the placeholder label  
+	.placeholderTextColor textColor of the placeholder label   
+	.atributedPlaceholer attributedString for the placeholder label.
+
+If you need more fine grained control over how the placeholder looks you can use the `.attributedPlaceholder`  
+**Note:** the size of the textview does not take into account a different font then the `.font` property
+
```
Comment (jatraiz) [inline README.md:35]: for the time being it is encouraged that these two fonts be equal

**Pull request merged**